builtin/logical/mssql/secret_creds.go

```diff
@@ -105,13 +104,13 @@ func (b *backend) secretCredsRevoke(ctx context.Context, req *logical.Request, d
 	// we need to drop the database users before we can drop the login and the role
 	// This isn't done in a transaction because even if we fail along the way,
 	// we want to remove as much access as possible
-	stmt, err := db.Prepare(fmt.Sprintf("EXEC master.dbo.sp_msloginmappings '%s';", username))
+	stmt, err := db.Prepare("EXEC master.dbo.sp_msloginmappings '?';")
```

Comment:
I feel like (and the other EXEC below) might've worked if you'd used `?` without quotes.  Can you try it that way please?

Reply:
Sure thing!

Reply:
Unfortunately, it doesn't actually work. :/

Reply:
I've been reading the microsoft docs and googling and I can't see any reason why this wouldn't work, or why EXEC should be special.  How does it fail?

Reply:
Well, take your pick:
 - with `'?'`: `"failed to revoke entry: resp: (*logical.Response)(nil) err: mssql: '?' is not a valid login or you do not have permission."`
 - with `?`: `"failed to revoke entry: resp: (*logical.Response)(nil) err: mssql: Incorrect syntax near '?'."`

I think what's happening is that prepare doesn't know how to work with stored procedures.